src/ImageSharp/Formats/Tiff/Utils/TiffUtils.cs

```diff
@@ -126,6 +151,27 @@ public static TPixel ColorScaleTo32Bit<TPixel>(ulong intensity, TPixel color)
             return color;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void UnPremultiplyRow<TPixel>(Span<Vector4> vectors, Span<TPixel> pixelRow, TPixel color)
```

Comment:
This looks like we're replicating existing functionality. We have `PixelOperations` for this.